milli/src/update/delete_documents.rs

```diff
@@ -523,6 +524,13 @@ impl<'t, 'u, 'i> DeleteDocuments<'t, 'u, 'i> {
             &self.to_delete_docids,
         )?;
 
+        // We delete the documents ids that are under the facet field id values.
+        remove_docids_from_facet_id_exists_docids(
```

Comment:
Should we rename this function since it no longer only remove the docids from `exists_docids`?

Reply:
I think the same

Reply:
Fixed, thank you! I just removed the `exists` part from the name 😄